definitions/npm/react-redux_v5.x.x/flow_v0.89.x-/test_connect.js

```diff
@@ -308,22 +342,33 @@ function testMapDispatchToPropsPassesActionCreators() {
   };
   const Connected = connect(null, mapDispatchToProps)(Com);
   <Connected passthrough={123}/>;
+  // Here Flow reports that `dispatch1` is missing from props,
+  // while the real error is still the missing passthroug property,
+  // giving the passthrough property fixes the wrongly titled error.
+  //$ExpectError no passthrough
```

Comment:
duplicate `//$ExpectError no passthrough` causes unnecessary errors. Once this is fixed, the PR is good to go!

Reply:
Sorry, did not get, what needs to be changed?

Reply:
@peter-leonov one line below there is second `//$ExpectError`: This one does not actually match anything.

Reply:
Now I see 🤦‍♀️ Fixed!